src/main/kotlin/uk/gov/justice/hmpps/offenderevents/services/HMPPSDomainEventsEmitter.kt

```diff
@@ -75,18 +75,18 @@ class HMPPSDomainEventsEmitter(
       else -> {
         val mapping = OffenderEvent.eventMappings[event] ?: return
         when (val offenderEvent = objectMapper.readValue(message, mapping)) {
-          is CaseNoteOffenderEvent -> toCaseNotePublished(offenderEvent).toListOrEmptyWhenNull()
-          is PrisonerReceivedOffenderEvent -> toPrisonerReceived(offenderEvent).toListOrEmptyWhenNull()
-          is PrisonerDischargedOffenderEvent -> toPrisonerReleased(offenderEvent).toListOrEmptyWhenNull()
-          is CellMoveOffenderEvent -> toCellMove(offenderEvent).toList()
-          is NonAssociationDetailsOffenderEvent -> toNonAssociationDetails(offenderEvent).toList()
-          is PersonRestrictionOffenderEventUpserted -> toPersonRestrictionUpserted(offenderEvent)
-          is PersonRestrictionOffenderEventDeleted -> toPersonRestrictionDeleted(offenderEvent)
-          is RestrictionOffenderEvent -> toRestriction(offenderEvent).toList()
-          is PrisonerActivityUpdateEvent -> toActivityChanged(offenderEvent).toList()
-          is PrisonerAppointmentUpdateEvent -> toAppointmentChanged(offenderEvent).toList()
-          is ImprisonmentStatusChangedEvent -> toImprisonmentStatusChanged(offenderEvent).toListOrEmptyWhenNull()
-          is SentenceDatesChangedEvent -> toSentenceDatesChanged(offenderEvent).toList()
+          is CaseNoteOffenderEvent -> offenderEvent.toDomainEvent().toListOrEmptyWhenNull()
```

Comment:
all other changes is to convert each function to extension function so "event" gets replaced with "this"

Reply:
could remove this entirely?